agent_deploy/kubernetes/sysdig-agent-daemonset-v1.yaml

```diff
@@ -69,8 +69,14 @@ spec:
             memory: 1024Mi
         readinessProbe:
           exec:
-            command: [ "test", "-e", "/opt/draios/logs/running" ]
+            command: [ "test", "-e", "/opt/draios/logs/startup" ]
```

Comment:
If this is used with an older version of the agent then the agent will never "start", correct?

Can we change the behavior of the "running" file instead of making the new file?

Reply:
You're right. The main concern about using the same `running` file, is that everybody may prevent its creation just by disabling the watchdog from the config file. I suppose this has been done for avoiding sending log lines to the backend  when an agent is running in debug mode (in which case the watchdog is not enabled)
Anyway I may work around it for having just one file